shard-operator/src/main/resources/application.properties

```diff
@@ -37,3 +37,10 @@ quarkus.log.console.json.fields.process-name.enabled=false
 quarkus.log.console.json.fields.logger-class-name.enabled=false
 quarkus.log.console.json.fields.thread-id.enabled=false
 quarkus.log.console.json.fields.sequence.enabled=false
+
+# prod defaults (not expected to change)
+secret.name=addon-smart-events-operator-parameters
```

Comment:
@vaibhavjainwiz Can we please namespace this property for consistency with all our other configuration in this file,  so `event-bridge.secret.name` 

Can you please also add a comment as to what the configuration parameter controls as it's not totally obvious.

Reply:
done